datastore/index.go

```diff
@@ -0,0 +1,85 @@
+// Datastore implements workers that work with the Redis Index and Cassandra Metric datastores
+package datastore
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"strconv"
+	"strings"
+
+	"github.com/jeffpierce/cassabon/config"
+	"github.com/jeffpierce/cassabon/middleware"
+	"gopkg.in/redis.v3"
+)
+
+// IndexMetricPath takes a metric path string, splits it, and sends it off to be recursively parsed and indexed.
+func IndexMetricPath(path string) {
+	// We've got a path!
+	config.G.Log.System.LogDebug("Indexer received path %s.")
+
+	// Split the metric into a slice of its parts
+	splitPath := strings.Split(path, '.')
+
+	// Initialize a Redis client.
```

Comment:
I think so.  I'll need to make sure that a connection pool is getting initialized rather than just a single connection.